integ/README.md

```diff
@@ -0,0 +1,57 @@
+# Integration tests
+
+The following workflow is how you can use these integration tests
+for testing your changes and verifying that new dependencies didn’t break the updater.
+It’s also similar to how the ECS updater MCM works, so it’s useful to go through it and should in total take less than 1 hour.
+
+1. You’ll want to set up a test ECS cluster. 
```

Comment:
There should be a blank line after this initial bullet item line, then all content part of 1 should be indented three spaces to make sure it gets formatted properly as item content when rendered.